src/assets/i18n/nl.json

```diff
@@ -1,8 +1,8 @@
 {
-  "CANCEL": "Annuleer",
+  "CANCEL": "Ik ga ze eerst opslaan",
```

Comment:
See my comment in the en.json file.

Reply:
Understood!
The short cut is not a neat cut:)
Are you willing to fix that?
I think it's quicker...